docs/items/automated_tests.md

```diff
@@ -0,0 +1,159 @@
+---
+id: automated_tests
+title: Automated Tests
+sidebar_position: 4
+---
+
+# Automated Tests
+
+Automated tests are designed to validate specific criteria and ensure compliance based on the user's input. CAT Services supports three types of automated tests:
+
```

Comment:
Guidelines on how to create an automated test 
The 3 tests are the results of the guidelines